scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -67,9 +71,21 @@ export function RelatedItemsDirective(
             scope.onCreated = (items: Array<IArticle>) => {
                 items.forEach((item) => {
                     scope.addRelatedItem(item);
+                    storage.setItem(item._id, scope.item._id);
                 });
             };
 
+            scope.$on('item:close', (evt, mainArticleId) => {
+                let parentItem = storage.getItem(mainArticleId) ? {_id: storage.getItem(mainArticleId)} : null;
```

Comment:
It'd be good not to call `getItem` twice and save a result from the first call into a variable.

It'd also avoid calling it `parentItem`, since it's only an ID. I'd write it like this:

```
scope.$on('item:close', (evt, mainArticleId) => {
    const itemId = storage.getItem(mainArticleId);

    if (itemId != null) {
        autosave.get({_id: itemId}).then((result) => {
...
```